client/store/userReducer.js

```diff
@@ -0,0 +1,39 @@
+const axios = require('axios')
+
+const SET_USER = 'SET_USER'
+
+const setUser = (user) => {
+    return {
+        type : SET_USER,
+        user
+    }
+}
+
+export const validateLogin = (loginInfo) =>{
+    return async (dispatch)=>{
+        console.log(loginInfo)
+       const {data} = await axios.post('/auth/validation/login',loginInfo,
+    {
+        credentials: 'same-origin'
+    })
+       dispatch(setUser({user:data}))
```

Comment:
do you guys have a linter set up?
you guys can use a package called "husky" which basically... whenever you commit your code, it'll automatically lint it all (according to the rules you set)
if you need help setting it up, ask one of us!